src/reynir_correct/main.py

```diff
@@ -142,20 +144,21 @@ class AnnResultDict(TypedDict):
 )
 
 parser.add_argument(
-    "suppress_suggestions",
+    "-suppress_suggestions",
```

Comment:
Is this definitely correct? Looks a bit suspicious. We tend to support only long-form argument names, in this case that would be --suppress_suggestions - but argparse optionally allows short-form arguments as well, with a single hyphen, which in this case would be -sss or just -s.

Reply:
Changed to "--suppress_suggestions" and "-sss" *(-s I think clashes with arguments from other sources).